apps/native/app/src/stores/preferences-store.ts

```diff
@@ -38,7 +38,7 @@ export interface PreferencesStore extends State {
 const availableLocales: Locale[] = ['en-US', 'is-IS', 'is-US', 'en-IS']
 
 const defaultPreferences = {
-  appearanceMode: 'automatic',
+  appearanceMode: 'light',
```

Comment:
~Let's keep the default preference as `automatic` in case we later implement dark mode again.